CRM/Core/Error.php

```diff
@@ -1035,12 +1035,17 @@ public static function isAPIError($error, $type = CRM_Core_Error::FATAL_ERROR) {
    *
    * @param $newMethod
    *   description of new method (eg. "buildOptions() method in the appropriate BAO object").
+   * @param $oldMethod
```

Comment:
@mattwire my only issue is this comment block - can we add the type (string) here?

Reply:
Yes! Done.